### PR TITLE
uuidgen improvements v2

### PR DESCRIPTION
**Pull request by karelzak**

### CODE REVIEW

**Comment by t-8ch** [comment]
Why 'C' over 'c'?

Otherwise LGTM

**Comment by karelzak** [comment]
I usually use capital letters for less commonly used (obscure) options :-) But I do not have a strong opinion in this case.

**Comment by t-8ch** [comment]
Me neither, I had the impression uppercase was used in case of conflicts.
But it's fine either way.

File: misc-utils/Makemodule.am [hunk 1]
```diff
@@ -104,7 +104,7 @@ usrbin_exec_PROGRAMS += uuidgen
 MANPAGES += misc-utils/uuidgen.1
 dist_noinst_DATA += misc-utils/uuidgen.1.adoc
 uuidgen_SOURCES = misc-utils/uuidgen.c
-uuidgen_LDADD = $(LDADD) libuuid.la
+uuidgen_LDADD = $(LDADD) libcommon.la libuuid.la
```
Comment (t-8ch) [inline misc-utils/Makemodule.am:107]: Missing the same for meson

**Comment by t-8ch** [comment]
FYI you can update the source branch of your PR and the PR will update with it.
There is no need to create a new PR.

**Comment by karelzak** [comment]
Nice. Thanks for the info. I thought it was possible to do a force push only.

**Pull request closed without merging**